scripts/ui_release.sh

```diff
@@ -11,7 +11,7 @@ files=("../LICENSE" "../CHANGELOG.md")
 workspaces=$(jq -r '.workspaces[]' < package.json)
 publish_workspaces=$(for workspace in ${workspaces}; do 
   # package "app" is private so we shouldn't try to publish it.
-  if [[ "${workspace}" != "app" ]]; then
+  if [[ "${workspace}" != "app" ]] && [[ "${workspace}" != "e2e" ]]; then
```

Comment:
It could be interesting to publish the page object models for re-use elsewhere in the future, but I don't want to dig into that with this initial setup.